packages/jupyter-ai/src/icons.ts

```diff
@@ -21,6 +22,11 @@ export const replaceCellIcon = new LabIcon({
   svgstr: replaceCellSvg
 });
 
+export const includeSelectionIcon = new LabIcon({
+  name: 'jupyter-ai::include-selection',
```

Comment:
Just curious, why do icons in jupyter-ai use double `::` in the identifiers?

Reply:
This was a pattern I created on a whim when building the UI initially. I figured the names should be namespaced to `jupyter-ai`, but didn't know the existing convention. So I just borrowed the C++ namespace resolution operator `::` as a delimiter here. 🤓 

Feel free to open a PR that alters these icon names to be more in-line with established conventions in Project Jupyter.